modules/virt-creating-data-volumes-using-storage-api.adoc

```diff
@@ -0,0 +1,41 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virtual_disks/virt-creating-data-volumes.adoc
+
+[id="virt-creating-data-volumes-using-storage-api_{context}"]
+
+= Creating data volumes using the Storage API
+
+When you create a data volume using the *storage* API, the CDI optimizes your allocation based on the attributes of the mounted volume. You only have to specify the data volume name, namespace, and the amount of storage you want to allocate.
```

Comment:
CDI optimizes your allocation based on the type of storage provided by the selected storage class.

Reply:
changed to:

_When you create a data volume using the *storage* API, the CDI optimizes your allocation based on the type of storage provided by your selected storage class._